powershell/setup-defender.ps1

```diff
@@ -48,12 +70,26 @@ if (!(IsAdministrator))
     }
 }
 
+
+
 # Set preference
 "Removing " + $path + " from Windows Defender's Eye"
-if (Get-Command Set-MpPreference -errorAction SilentlyContinue) {
+
+
+# https://msdn.microsoft.com/en-us/library/windows/desktop/ms724832%28v=vs.85%29.aspx
+$version = [Environment]::OSVersion.Version
+
+if ($version.Major -eq 6 -AND $version.Minor -gt 1) {
```

Comment:
This looks good, but we need to account for Windows 10 and above (major 10, minor currently 0). This should work:
```
if (($version.Major -eq 6 -AND $version.Minor -gt 1) -or ($version.Major -gt 6)) {
```